crates/compiler/build/src/link.rs

```diff
@@ -410,17 +413,38 @@ pub fn build_c_host_native(
         .args(sources)
         .args(&["-o", dest]);
     if let Some(shared_lib_path) = shared_lib_path {
-        command.args(&[
-            shared_lib_path.to_str().unwrap(),
-            &bitcode::get_builtins_host_obj_path(),
-            "-fPIE",
-            "-pie",
-            "-lm",
-            "-lpthread",
-            "-ldl",
-            "-lrt",
-            "-lutil",
-        ]);
+        match target.operating_system {
+            OperatingSystem::Windows => {
+                // just use zig as a C compiler
+
+                // I think we only ever have one C source file in practice
+                assert_eq!(sources.len(), 1);
+
+                return build_zig_host_native(
+                    env_path,
+                    env_home,
+                    &format!("-femit-bin={}", dest),
+                    sources[0],
+                    find_zig_str_path().to_str().unwrap(),
+                    "x86_64-windows-gnu",
```

Comment:
gnu?! Surprised to see that mentioned in Windows stuff. Do they have a libc for native Windows? Just curious.